tests/project_helpers.py

```diff
@@ -100,7 +102,7 @@ def get(self, cache: pytest.Cache) -> Path:
 
     @property
     def cache_key(self) -> str:
-        return f"fawltydeps/{self.tarball_name()}"
+        return os.path.join("fawltydeps", f"{self.tarball_name()}")
```

Comment:
With `pathlib`, I'm trying to deprecate `os.path` as much as possible... 😅 

```suggestion
        return Path("fawltydeps", self.tarball_name())
```
plus `from pathlib import Path` at the top.